modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,117 @@
+//You can upgrade local Mule instances to the latest available version or to a particular Mule distribution that you have previously downloaded and unpacked.
+You can also use the Mule upgrade tool to update Mule instances located in different folders.
+
+As part of the upgrade process, the Mule upgrade tool also updates the Runtime Manager agent version if it is installed in your current Mule instance.
+
+== Before You Begin
+
+Before starting your upgrade, ensure that you have:
+
+* A currently operational Mule 4 instance, in _stopped_ status to prepare for the upgrade
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory
++
+If the Mule upgrade tool is not bundled with your current Mule distribution, download the tool from the https://help.mulesoft.com/s/[Support Portal^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.
+//* The new Mule runtime engine distribution you want to upgrade to
+* The latest Mule runtime engine distribution, downloaded and unpacked in your system
++
+You can download Mule distributions from the https://help.mulesoft.com/s/[Support Portal^].
+* At least 2 GB of available disk space on the file system and access privileges to install the new Mule distribution
+//Confirm if the tool is downloaded from the Support portal or help center, and update the link.
+
+//This feature is not yet available
+////
+== Upgrade Mule to the Latest Version
+
+To upgrade your Mule instance to the latest version:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade` command:
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade
+```
+////
+
+== Upgrade Mule
+
+To upgrade your Mule instance to the latest version:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade -n` command, specifying the directory of the latest Mule distribution as an argument:
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade -n ./<newMuleDistributionPath>
+
+```
+
+== Roll Back an Upgrade
+
+The Mule upgrade tool generates a backup of your existing Mule instance before starting the upgrade process. The backup is stored in the `$MULE_HOME/backup` folder in case you want to use the tool to restore your upgraded Mule instance to its previous state.
+
+To perform a rollback:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `rollback` command:
+```
+$MULE_HOME/tools/upgrade-tool rollback
+```
+
+== Mule Upgrade Tool Commands and Options
+
+The following table lists and describes the commands and options you can use when running the Mule upgrade tool:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+|===
+|Command | Description
+| `upgrade`
+| Upgrades the local Mule instance to the latest available version. +
+The following are the supported options for this command:
+
+[%header%autowidth.spread,cols=".^a,.^a"]
+!===
+  ! Options ! Description
+  ! `-n <newMuleDistributionPath>`
```

Comment:
We could show the long option of the parameters and the help parameter as it is shown in the upgrade tool readme https://github.com/mulesoft/mule-runtime-upgrade-tool/blob/master/README.md#usage